webview.h

```diff
@@ -1425,6 +1428,16 @@ inline bool enable_dpi_awareness() {
   return true;
 }
 
+inline void scale(HWND m_window, int *width, int *height) {
```

Comment:
Yes - that looks good. My inclination would be to be slightly more efficient and handle both values at once.  A true or false could be  returned - maybe something like this:
```
inline bool scale_with_window_dpi(HWND window, int *width, int *height) {
  constexpr const int default_dpi = 96;
  auto user32 = native_library(L"user32.dll");
  if (auto fn = user32.get(user32_symbols::GetDpiForWindow)) {
    int dpi = static_cast<int>(fn(window));
    *width = (*width * dpi) / default_dpi;
    *height = (*height * dpi) / default_dpi;
    return true;
  }
  return false;
}
```

But, certainly either way will work and i'm happy to go the route you propose -

Reply:
It might be a little bit wasteful although I suspect it won't really be noticeable as long as `user32.dll` is already loaded—which it should be. Another approach could be to just return the scale factor as a `double`, e.g. `1.25` for 125% scaling.

Reply:
I was trying to keep it all integer math with the parentheses e.g. (*width * dpi) / ...

No, certainly won't be noticeable, i'm just kind of an efficiency freak :)

Code it up your way then? Fine by me -

Reply:
Yes, I would like to suggest using my initial approach to avoid in/out parameters in C++ code. I admit it could be more efficient but at the same time other areas of the library will benefit more from optimizations. Just keeping it simple will do here.